### ISSUE TITLE
Inconsistent shield icons confuse users during verif.

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#14805]
We should probably change the prompt to be "Is the other user showing a tick?" in order to be more cross-client compatible, or something rather than "the same shield".

<img width="987" alt="Screenshot 2020-07-28 at 22 40 30" src="https://user-images.githubusercontent.com/1294269/88725108-7e39d000-d123-11ea-99ab-f9640b070e7b.png">

**Comment by turt2live** [comment]
also we should probably be using the same asset in both places - looks like the login stages are using the older one.

**Comment by brandonhaslegs** [comment]
Came here to report this. The assets are different, so I almost didn't click "Yes."

**Comment by DC4JG** [comment]
Just linked my cheap china android device and because the shields were so different in art style, especially due to white border on dark mode I got suspicious and started searching for how it should or shouldn't look on the internet. Found this thread and just wanted to report, that the very different art styles are confusing.

I almost thought there could be an emoji like verification with different shield borders, symbols, forms, sizes, ... the developer in me knew that this would be silly (I used the QR code) but the paranoid security guy in me was upset enough to not immediately accept it and double check just in case.

**Comment by tmarti2** [comment]
I just answered "no" to this question because they are cleary not identical and was surprised to read "canceling" afterward...
I also thought it was a emoji  like verification.

**Comment by GirardR1006** [comment]
Agreed, on dark mode one shield has a white border while the other has none, which is clearly error-prone.
